packages/grid-pro/theme/material/vaadin-grid-pro-styles.js

```diff
@@ -16,8 +16,7 @@ registerStyles(
 
     [part~='row'] > [part~='editable-cell']:hover,
     [part~='row'] > [part~='editable-cell']:focus {
-      background-color: var(--material-grid-pro-editable-cell-hover-background-color, rgba(0, 0, 0, 0.04));
-      background-clip: padding-box;
+      background: var(--material-background-color) linear-gradient(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.08));
```

Comment:
Since Material theme has a row highlight color, changing the background here to a solid color needs to compensate for that. The row background is `rgba(0, 0, 0, 0.04)`, so adding another `rgba(0, 0, 0, 0.04)` should result in `rgba(0, 0, 0, 0.08)` (right?).

Reply:
So it seems (checked manually in Figma). I was expecting it not to, but apparently the math checks out in this case. I mean, if you stack ten `rgba(0, 0, 0, 0.1)`s on top of each other, they don't result in `rgba(0, 0, 0, 1)`.